book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_reader :rentals
```

Comment:
- [ ] It would be better if we set our `rentals` to both getter and setter so that we can perform both read and write operation instead of only reading the rentals value. ***HINT***: `attr_accessor`